pinot-spi/src/main/java/org/apache/pinot/spi/config/table/ingestion/StreamIngestionConfig.java

```diff
@@ -34,6 +34,9 @@ public class StreamIngestionConfig extends BaseJsonConfig {
   @JsonPropertyDescription("All configs for the streams from which to ingest")
   private final List<Map<String, String>> _streamConfigMaps;
 
+  @JsonPropertyDescription("Whether to use column major mode when creating the segment.")
+  private boolean _columnMajorSegmentBuilderEnabled;
```

Comment:
Both are checked in the `RealTimeSegmentDataManager` constructor: if there's a Stream Ingestion Config section then it uses that to check for the Segment Builder mode and if there isn't then it checks the the old configuration section for the flag.

Reply:
Since this is a newly added temporary flag, don't see much value supporting it in 2 different places. Let's just remove it from here and only keep the one in `IndexingConfig` for simplicity